vars/buildPlugin.groovy

```diff
@@ -68,25 +68,17 @@ def call(Map params = [:]) {
                                     readFile('.mvn/extensions.xml').contains('git-changelist-maven-extension')
                             if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
                                 if (isUnix()) {
-                                    sh(script: 'git clean -xffd > /dev/null 2>&1',
-                                       label: 'Clean for incrementals',
-                                       returnStatus: true) // Ignore failure if CLI git is not available
+                                    sh 'git clean -xffd || echo "Git CLI may not be available"'
                                 } else {
-                                    bat(script: 'git clean -xffd 1> nul 2>&1',
-                                        label: 'Clean for incrementals',
-                                        returnStatus: true) // Ignore failure if CLI git is not available
+                                    bat 'git clean -xffd || VER>NUL' // Ignore failure if CLI git is not available
                                 }
                             }
 
                             if (gitDefaultBranch) {
                                 if (isUnix()) {
-                                    sh(script: "git config --global init.defaultBranch '${gitDefaultBranch}'",
-                                       label: 'Set default branch for git',
-                                       returnStatus: true) // Ignore failure if CLI git is not available
+                                    sh "git config --global init.defaultBranch '${gitDefaultBranch}' || :" // Ignore failure if CLI git is not available
```

Comment:
Seems like you'll want the same message if this command fails as if the `git clean` fails.
```suggestion
                                    sh "git config --global init.defaultBranch '${gitDefaultBranch}' || echo Git CLI may not be available" // Ignore failure if CLI git is not available
```

Reply:
Thanks! I was planning to find « right way » to do it in windows as well, and got carried away with the CVE so put it back to draft. The suggestions helps and come handy!

Reply:
Updated WDYT?